src/event_trigger.c

```diff
@@ -162,26 +174,48 @@ make_event_trigger_drop_view(char *view_name, char *schema)
 }
 
 static EventTriggerDropSchema *
-make_event_trigger_drop_schema(char *schema)
+make_event_trigger_drop_schema(const char *schema)
 {
 	EventTriggerDropSchema *obj = palloc(sizeof(EventTriggerDropSchema));
 
 	*obj = (EventTriggerDropSchema){
-		.obj = { .type = EVENT_TRIGGER_DROP_SCHEMA },
+		.obj = {
+			.type = EVENT_TRIGGER_DROP_SCHEMA,
+		},
 		.schema = schema,
 	};
+
 	return obj;
 }
 
 static EventTriggerDropTrigger *
-make_event_trigger_drop_trigger(char *trigger_name, char *schema, char *table)
+make_event_trigger_drop_trigger(const char *trigger_name, const char *schema, const char *table)
 {
 	EventTriggerDropTrigger *obj = palloc(sizeof(EventTriggerDropTrigger));
 
-	*obj = (EventTriggerDropTrigger){ .obj = { .type = EVENT_TRIGGER_DROP_TRIGGER },
-									  .trigger_name = trigger_name,
-									  .schema = schema,
-									  .table = table };
+	*obj = (EventTriggerDropTrigger){
+		.obj = {
+			.type = EVENT_TRIGGER_DROP_TRIGGER,
+		},
+		.trigger_name = trigger_name,
+		.schema = schema,
+		.table = table
+	};
+
+	return obj;
+}
+
+static EventTriggerDropForeignServer *
+make_event_trigger_drop_foreign_server(const char *server_name)
+{
+	EventTriggerDropForeignServer *obj = palloc(sizeof(EventTriggerDropForeignServer));
```

Comment:
Will it be more safe to call `palloc0`?
```suggestion
	EventTriggerDropForeignServer *obj = palloc0(sizeof(EventTriggerDropForeignServer));
```